src/utils.py

```diff
@@ -3,11 +3,40 @@
 
 """A collection of utility functions."""
 
+import logging
+import shutil
 from bz2 import open as bopen
 from gzip import open as gopen
 from lzma import open as lopen
 from os import readlink
 from pathlib import Path
+from typing import Set
+
+logger = logging.getLogger(__name__)
+
+
+def clean_dists(path: Path) -> None:
+    """Clean dists for mirror path."""
+    mirror_path = path / "mirror"
+    for dists in mirror_path.rglob("**/dists"):
+        shutil.rmtree(dists)
+        logger.info("Removed %s", dists)
+
+
+def clean_packages(packages: Set[Path]) -> bool:
+    """Clean up packages."""
+    logger.info("Cleaning up unreferenced packages")
+    result = True
+    for package in packages:
+        try:
+            package.unlink()
+            logger.info("Removed %s", package)
```

Comment:
I think cleanups will easily encompass hundreds of packages, so I'd suggest to limit this to debug severity.
```suggestion
            logger.debug("Removed %s", package)
```

Reply:
I agree and I used debug also in `clean_dists` function.